doc/configuration.md

```diff
@@ -399,6 +399,7 @@ volume.size                     | string    | appropriate driver
 volume.zfs.remove\_snapshots    | bool      | zfs driver                        | false             | Remove snapshots as needed
 volume.zfs.use\_refquota        | bool      | zfs driver                        | false             | Use refquota instead of quota for space.
 zfs.pool\_name                  | string    | zfs driver                        | name of the pool  | Name of the zpool
+zfs.clone\_copy                 | bool      | zfs driver                        | false             | Whether copied containers require dependent snapshots in the zfs storage pool.
```

Comment:
How about just describing it as "Whether to use ZFS lightweight clones rather than full dataset copies"?

That hints that setting this to false would cause disk usage to go up.